### DISCUSSION TITLE
lldpcli when attempting to configure  agent-type on OSX returns Bad file descriptor

### DISCUSSION THREAD

**Discussion by juacostab** [lldpd/lldpd#471] [Q&A]
I tried to set the `Agent-Type` to  `nearest-customer-bridge` with `lldpcli` on my OSX 11.5.1 (BigSur), but fails as shown next

```
$ sudo lldpcli configure lldp agent-type nearest-customer-bridge
2021-08-18T12:03:14 [WARN/lldpctl] cannot get lock on /var/run/lldpd.socket: Bad file descriptor
```

I did all troubleshooting I could and was not able to spot what might be wrong. Compared with my Linux deployment and file permissions look ok also

`OSX`
```
sudo ls -l /var/run/lldpd.socket
Password:
srwxrwx---  1 _lldpd  _lldpd  0 Aug 18 11:44 /var/run/lldpd.socket
```

`Linux`
```
~$ ls -l /var/run/lldpd.socket 
srwxrwx--- 1 _lldpd _lldpd 0 Aug 17 17:18 /var/run/lldpd.socket
```

**Comment by vincentbernat** [comment]
I think this is #445. You can try master or the "package" link on the following page: https://github.com/lldpd/lldpd/actions/runs/1093146775

**Reply by juacostab** [reply]
Thanks a lot I will try that. I will report back

**Reply by juacostab** [reply]
@vincentbernat Thanks a lot. I took the package you indicated on the link above and that solved the issue
